substrate/primitives/io/src/lib.rs

```diff
@@ -1272,6 +1272,24 @@ pub trait Crypto {
 		Ok(res)
 	}
 
+	/// Recover an Ethereum address from a SECP256k1 ECDSA signature.
+	///
+	/// - `sig` is passed in RSV format. V should be either `0/1` or `27/28`.
+	/// - `msg` is the blake2-256 hash of the message.
+	///
+	/// Returns `Err` if the signature is bad, otherwise the 20-bytes address.
+	///
+	/// It can handle non-standard overflowing signatures.
+	fn secp256k1_ecdsa_recover_address(
```

Comment:
so you will later add some feature flags in there to define a custom logic for the anvil node?